src/pages/builder/index.tsx

```diff
@@ -48,7 +105,7 @@ export default function BuilderPage() {
         <EditForm />
       </div>
       <div style={{ border: '1px solid red', width: '100%', height: '100%' }}>
-        <CvViewer cv={cvData} />
+        <CvViewer cv={cvDataFromWizard} />
       </div>
```

Comment:
pay attention that you get cv from right context

Reply:
I deliberately set this like this, to show the results of the Wizard

Reply:
I used data from the Wizard as a default values for CVContext, let's leave CVContext too